src/pharmpy/tools/modelfit/tool.py

```diff
@@ -20,7 +20,7 @@ def create_workflow(
     n : int
         Number of models to fit. This is only used if the tool is going to be combined with other tools.
     tool : str
-        Which tool to use for fitting. Currently, 'nonmem' or 'nlmixr' can be used.
+        Which tool to use for fitting. Currently, 'nonmem', 'nlmixr', 'rxode' can be used.
```

Comment:
This seems to indicate that rxode can fit, but it can really only evaluate (predict).

Reply:
This was done in order to be able to run rxode models in the same way as the for the other tools (instead of having to use execute model and define your own database).
Should I remove rxode support from this function?

Reply:
No, I didn't mean that support should be removed. Rather that the docstring should clarify, but it might be difficult to do because the tool is named `Modelfit`, which is starting to become a bit of a misnomer in itself. So keep this for now.